### ISSUE TITLE
Warn when input contains legacy formatting codes

### ISSUE THREAD

**Issue by kezz** [KyoriPowered/adventure-text-minimessage#189]
Perhaps even just throwing an exception would be nice

**Comment by Proximyst** [comment]
https://github.com/KyoriPowered/adventure/blob/d92fcb2ded711e8d32679b5ef38aa624bb09ea5d/api/src/main/java/net/kyori/adventure/text/TextComponentImpl.java#L40

Isn't this already done?

**Comment by kezz** [comment]
Yes, but this is disabled by default. MiniMessage could easily have this enabled by default as the reasons why it was disabled by default for the API aren't present when constructing components with MiniMessage.

**Comment by MiniDigger** [comment]
should only throw exception in strict mode (if thats still a thing? dont remember)
